tests/posix/sem/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
```

Comment:
Split your patch in 2: adding semaphore support and then adding a test for it.

Reply:
Done